src/screens/fareResults.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { ScreenNavigationProps } from '../routes';
+import { config } from '../config';
+import FlatListComponent from '../components/flatList';
+import { useEffect } from 'react';
+import { FareResponse, Journey } from '../model/fareResponseClass';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#eee',
+  },
+  text: {
+    paddingBottom: 24,
+  },
+});
+
+type FareResultsProps = ScreenNavigationProps<'FareResults'>;
+
+const getTrainFares = async (
+  departStation: string | null,
```

Comment:
similar to other places, it doesn't make sense to allow null here - you should force it to be provided to work